app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -28,15 +30,24 @@ class Uploader extends \Magento\Backend\Block\Widget
     protected $_fileSizeService;
 
     /**
+     * @var \Magento\Framework\Json\EncoderInterface
+     */
+    protected $_jsonEncoder;
```

Comment:
please remove `_` and make it private

Reply:
It's the way it's done at \Magento\Downloadable\Block\Adminhtml\Catalog\Product\Edit\Tab\Downloadable\Links::getConfigJson, but I'll change it.

Reply:
Done